LandSandBoat/server
# [sql] [lua] Sing Horde Lullaby II to your enemies!

**_I affirm:_**

## What does this pull request do?

* Added job/level (BRD 92) to Horde Lullaby II in spell_list.sql.
* Corrected base spell duration in horde_lullaby_ii.lua.

Source for the above: BG-Wiki

## Steps to test these changes

1. ``!addallspells``
2. ``!changejob brd 91``
    * Notice that you cannot even see the spell **Horde Lullaby II** in your spell list.
3. ``!changejob brd 92``
    * Notice that you now can see/cast the spell **Horde Lullaby II**, hopefully making lots of meanie meanie mobs go sleepy sleep for the correct base duration.

The above steps assume that you already cherry-picked this commit (or waited for it to be merged into base branch and then synced with base branch) before attempting to test the above.

No mobs were harmed in the testing of this PR, only gently, softly, soothingly put to sleep.
